Categories/UITableViewCell+TCAdditions.h

```diff
@@ -36,34 +37,34 @@
  *
  *  @return A reuse identifier that is equal to the class name.
  */
-+ (NSString *)tc_reuseIdentifier;
++ (NSString * __tc_null_unspecified)tc_reuseIdentifier;
 
 /**
  *  Returns a string equal to the class name.
  *
  *  @return A string equal to the class name.
  */
-+ (NSString *)tc_nibName;
++ (NSString * __tc_null_unspecified)tc_nibName;
 
 /**
  *  Returns a string that is equal to the class name with "_iPad" appended to it.
  *
  *  @return A string that is equal to the class name with "_iPad" appended to it.
  */
-+ (NSString *)tc_iPadNibName;
++ (NSString * __tc_null_unspecified)tc_iPadNibName;
```

Comment:
Personal preference: Deprecate/remove the iPad versions, as they shouldn't be needed anymore